Header header

int16 pos_x
int16 pos_y
int16 pos_z

